agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -758,19 +757,24 @@ func MetadataFromContainer(dockerContainer *docker.Container) DockerContainerMet
 			return DockerContainerMetadata{Error: apierrors.NamedError(err)}
 		}
 	}
+
+	createdTime, _ := time.Parse(time.RFC3339Nano, dockerContainer.Created)
```

Comment:
i think fsouza/go-dockerclient uses the default Unmarshaler interface for time.Time, which uses time.RFC3339 (https://golang.org/src/time/time.go?s=36799:36846#L1215). so you should probably use the same format?